include/micm/util/cuda_vector_matrix.cuh

```diff
@@ -0,0 +1,15 @@
+#include <vector>
+
+namespace micm
+{
+  namespace cuda
+  {
+    int MallocVector(double *d_data, std::size_t num_elements);
+
+    int FreeVector(double *d_data);
+
+    int CopyToDevice(double *d_data, const double *h_data, std::size_t num_elements);
+
+    int CopyToHost(double *d_data, double *h_data, std::size_t num_elements);
```

Comment:
maybe add a short description for each of these function in doxygen format? it's pretty clear from the names what they do, and we haven't been very consistent with adding these comments, but it would be nice to move in this direction

Reply:
Documentation added!